<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="url_LE">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="crawl_item_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>단독상품 가져오기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="crawl_store_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>스토어 전체 가져오기</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="data_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1561</width>
      <height>761</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>130</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>10</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>90</number>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>상품</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>상품명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>정가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>할인가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>옵션</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>옵션별가격</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>추가옵션</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>옵션별가격</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>배송비</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>반송비</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>대표imgURL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>추가imgURL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>상세내용</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>태그</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>원산지</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>수입사</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>as번호</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>as안내</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>브랜드</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>제조사</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>제품상태</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>카테고리id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>제조국</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="export_data_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>저장하기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>URL :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>가져온 상품수:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>수집안된 상품:</string>
    </property>
   </widget>
   <widget class="QLabel" name="success_num">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="fail_num">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retry_btn">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>10</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>실패물품 다시시도</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkall_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>모두 선택</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>선택 삭제</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>1177</x>
      <y>40</y>
      <width>391</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>소요시간</string>
    </property>
   </widget>
   <widget class="QLabel" name="running_time">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>40</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0분00초</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uncheckall_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>모두 해제</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_num">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>검색수:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>검색가능한 최대 갯수:</string>
    </property>
   </widget>
   <widget class="QLabel" name="max_search_num">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_start_btn">
    <property name="geometry">
     <rect>
      <x>882</x>
      <y>40</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>검색시작</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>예상시간</string>
    </property>
   </widget>
   <widget class="QLabel" name="expect_time">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>40</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0분00초</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_category_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>카테고리 검색</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_btn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>등록하기</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
